templates/sshd.template.yml

```diff
@@ -1 +1,4 @@
-# This file is deprecated; you can remove it from your app.yml
+run:
+  - exec: echo "The sshd.template.yml file is deprecated; you can remove it from your containers/app.yml file."
+  # TODO(2020-09-01): change to "exit 1"
+  - exec: bash -c "exit 0"
```

Comment:
I guarantee nobody is going to look at the output when the build succeeds, may as well just tell them *now* to remove it.

Reply:
Set a hard cutoff date for this September.